### PR TITLE
Create water-sanitation-and-hygiene-for-development.csl

### PR DESCRIPTION
**Pull request by jeccak**
New style, for this academic journal:  http://washdev.iwaponline.com/content/instructions-authors

I modified this to create a style for the above linked journal, and wanted to share it with others.  Thanks!

### CODE REVIEW

File: water-sanitation-and-hygiene-for-development.csl [hunk 1]
```diff
@@ -0,0 +1,618 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Water Sanitation and Hygiene for Development</title>
+    <title-short>WASHEV</title-short>
+    <id>http://www.zotero.org/styles/water-sanitation-and-hygiene-for-development</id>
+    <link href="http://www.zotero.org/styles/water-sanitation-and-hygiene-for-development" rel="self"/>
+    <link href="http://owl.english.purdue.edu/owl/resource/560/01/" rel="documentation"/>
```
Comment (adam3smith) [inline water-sanitation-and-hygiene-for-development.csl]: We'd want the documentation for the journal style linked here.

Comment (adam3smith) [inline water-sanitation-and-hygiene-for-development.csl]: Also, add a template link to the style you based this on (apa I take it?) i.e.
`<link href="http://www.zotero.org/styles/apa" rel="template"/>`

File: water-sanitation-and-hygiene-for-development.csl [hunk 2]
```diff
@@ -0,0 +1,618 @@
...
+    <author>
```
Comment (adam3smith) [inline water-sanitation-and-hygiene-for-development.csl]: with a template link set, you can add yourself as the style's author and remove all other authors and contributors.

File: water-sanitation-and-hygiene-for-development.csl [hunk 3]
```diff
@@ -0,0 +1,618 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
```
Comment (adam3smith) [inline water-sanitation-and-hygiene-for-development.csl]: add a default locale for the style; I'd assume `default-locale="en-US"`

File: water-sanitation-and-hygiene-for-development.csl [hunk 4]
```diff
@@ -0,0 +1,618 @@
...
+    <contributor>
+      <name>Sebastian Karcher</name>
+    </contributor>
+<contributor>
+      <name>Jessica Kaminsky</name>
+      <uri>http://www.heterogeneous-engineering.org</uri>
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="engineering"/>
+    <category field="generic-base"/>
```
Comment (adam3smith) [inline water-sanitation-and-hygiene-for-development.csl]: remove the `generic-base` field

File: water-sanitation-and-hygiene-for-development.csl [hunk 5]
```diff
@@ -0,0 +1,618 @@
...
+        <multiple>trans.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="container-contributors">
+    <choose>
+      <if type="chapter paper-conference entry-dictionary entry-encyclopedia" match="any">
+        <group delimiter=", " suffix=", ">
+          <names variable="container-author" delimiter=", ">
+            <name and="symbol" initialize-with=". " delimiter=", "/>
```
Comment (adam3smith) [inline water-sanitation-and-hygiene-for-development.csl]: had a quick look at the author instructions and you'll want to use `delimiter-precedes-last="never"` for all lines starting with `<name` (i.e. there's never a comma before the ampersand).

File: water-sanitation-and-hygiene-for-development.csl [hunk 6]
```diff
@@ -0,0 +1,618 @@
...
+    <choose>
+      <if variable="original-date">
+        <group prefix="(" suffix=")" delimiter=" ">
+          <text value="Original work published"/>
+          <date variable="original-date" form="text"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <citation et-al-min="6" et-al-use-first="1" et-al-subsequent-min="3" et-al-subsequent-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" collapse="year" givenname-disambiguation-rule="primary-name">
```
Comment (adam3smith) [inline water-sanitation-and-hygiene-for-development.csl]: Style says to always use et-al after 2, i.e. just do ` et-al-min="3" et-al-use-first="1"` delete the subsequents

File: water-sanitation-and-hygiene-for-development.csl [hunk 7]
```diff
@@ -0,0 +1,618 @@
...
+    </sort>
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group delimiter=" ">
+        <text macro="author-short"/>
+        <text macro="issued-year"/>
+        <text macro="citation-locator"/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography hanging-indent="true" et-al-min="8" et-al-use-first="6" et-al-use-last="true" entry-spacing="0" line-spacing="2">
```
Comment (adam3smith) [inline water-sanitation-and-hygiene-for-development.csl]: no et al in bibliography. Delete the et-al settings here.

File: water-sanitation-and-hygiene-for-development.csl [hunk 8]
```diff
@@ -0,0 +1,618 @@
...
+      <else-if type="post-weblog webpage" match="none">
+        <group delimiter=", ">
+          <choose>
+            <if variable="event version" type="speech" match="none">
+              <text variable="genre"/>
+            </if>
+          </choose>
+          <choose>
+            <if type="article-journal article-magazine" match="none">
+              <group delimiter=": ">
```
Comment (adam3smith) [inline water-sanitation-and-hygiene-for-development.csl]: this should be a comma

File: water-sanitation-and-hygiene-for-development.csl [hunk 9]
```diff
@@ -0,0 +1,618 @@
...
+      </if>
+      <else-if type="dataset" match="any">
+        <text value="Data set"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <choose>
+      <if type="report" match="any">
+        <group delimiter=": ">
```
Comment (adam3smith) [inline water-sanitation-and-hygiene-for-development.csl]: comma here, too.

**Comment by rmzelle** [comment]
@jeccak, let us know if you need help.

(I just noticed that this pull request is for the style requested in https://github.com/citation-style-language/styles/issues/1834 (by @jeccak))

**Comment by rmzelle** [comment]
Closing because of inactivity. @jeccak, you can still make the requested changes; if you do that and leave a comment, we'll take another look.

**Pull request closed without merging**